internal/pkg/install/options.go

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package install
+
+// Option controls generate options specific to input generation.
+type Option func(o *Options) error
+
+// WithExistingImage specifies whether to pull the installer image during an install
+func WithExistingImage() Option {
```

Comment:
nit: Should we make this `WithPullImage` if we want to use `PullImage` as the field? If we want `WithExistingImage`, maybe we should make the field `ExistingImage` or something like that. At the end of the day, I don't care either way, but I feel like there is some double negation type thinking this forces me to do since `WithExistingImage` sort of inverts what the field `PullImage` makes you think. Can you tell I'm confused? :D